src/src/auth/auth.service.ts

```diff
@@ -339,7 +339,81 @@ export class AuthService {
 
     }
 
+    public async isUserExist(req: any) {
```

Comment:
does this validate phone, email, aadhaar all 3?

Reply:
only phone